lib/nmatrix/math.rb

```diff
@@ -112,6 +112,58 @@ def invert
   end
   alias :inverse :invert
 
+  # call-seq:
+  #     invert_exact! -> NMatrix
+  #
+  # Calulates inverse_exact of a matrix of size 2 or 3.
+  # Only works on dense matrices.
+  #
+  # * *Raises* :
+  #   - +DataTypeError+ -> cannot invert an integer matrix in-place.
+  #   - +NotImplementedError+ -> cannot find exact inverse of matrix with size greater than 3  #
+  def invert_exact!
```

Comment:
I think we should probably alias this as `invert_exactly!` and name it `exact_inverse!` instead. `invert_exact!` is not a good name.

FYI, you can make these changes and then do a `git commit -a --amend` followed by a `git push --force` if you want to update the pull request instead of having to open a brand new one every time.